pkg/apis/pipeline/v1/pipeline_validation_test.go

```diff
@@ -2599,6 +2640,63 @@ func TestContextInvalid(t *testing.T) {
 			Also(apis.ErrGeneric(`non-existent variable in "$(context.pipeline.missing-foo)"`, "value")).
 			Also(apis.ErrGeneric(`non-existent variable in "$(context.pipelineRun.missing-foo)"`, "value")).
 			Also(apis.ErrGeneric(`non-existent variable in "$(context.pipelineTask.missing-foo)"`, "value")),
+	}, {
+		name: "invalid string context variable for pipeline in include matrix",
+		tasks: []PipelineTask{{
+			Name:    "bar",
+			TaskRef: &TaskRef{Name: "bar-task"},
+			Params: []Param{{
```

Comment:
please remove params from these test cases

Reply:
@EmmaMunley was this comment addressed?

Reply:
Yes it should have been